### ISSUE TITLE
H2 Console: with Chrome and Safari, resizing the table/query frame doesn't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#225]
```
What steps will reproduce the problem?

1. Use Google Chrome browser to connect to a database 
2. Try resizing tables list frame or query input frame


What is the expected output? What do you see instead?

It should be possible to resize frames as it's useful when a large query is 
being edited or table names are longer!

What version of the product are you using? On what operating system, file
system, and virtual machine?

H2-1.2.140, Ubuntu-10.04

Do you know a workaround?

Use Firefox instead!

How important/urgent is the problem for you?

Not so urgent

In your view, is this a defect or a feature request?

It is a defect.

Please provide any additional information below.

By changing the frameborder to 1 (from 0), resizing works, but the view doesn't 
look good. So there may be other workarounds for this!

```

Original issue reported on code.google.com by `prashant...@gmail.com` on 21 Aug 2010 at 7:25

**Comment by GoogleCodeExporter** [comment]
```
Hi,

What version of Chrome do you use? Could you try with the newest version? For 
me it works (I use Chrome 5.0.375.127 on Mac OS X 10.6).
```

Original comment by `thomas.t...@gmail.com` on 24 Aug 2010 at 1:04

**Comment by GoogleCodeExporter** [comment]
```
I'm using Chrome 6.0.472.36 on Ubuntu-10.04 and 6.0.472.41 on Windows XP. In 
both the systems, resizing doesn't work.
```

Original comment by `prashant...@gmail.com` on 25 Aug 2010 at 2:30

**Comment by GoogleCodeExporter** [comment]
```
Sorry, I misunderstood what the problem is... Yes, I can reproduce it on my 
machine. I will try to fix it in the next release (not sure yet how).
```

Original comment by `thomas.t...@gmail.com` on 25 Aug 2010 at 5:25
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `thomas.t...@gmail.com` on 25 Aug 2010 at 6:46
- Changed title: **H2 Console : with Chrome and Safari, resizing menu/query frame doesn't work**

**Comment by GoogleCodeExporter** [comment]
Original comment by `thomas.t...@gmail.com` on 25 Aug 2010 at 6:46
- Changed title: **H2 Console: with Chrome and Safari, resizing menu/query frame doesn't work**

**Comment by GoogleCodeExporter** [comment]
```
Will be fixed in the next release.
```

Original comment by `thomas.t...@gmail.com` on 25 Aug 2010 at 9:00
- Changed title: **H2 Console: with Chrome and Safari, resizing the table/query frame doesn't work**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.2.142
```

Original comment by `thomas.t...@gmail.com` on 31 Aug 2010 at 6:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot, it is much easier to use Console now!
```

Original comment by `prashant...@gmail.com` on 31 Aug 2010 at 6:38
